app/javascript/packs/utils/honeybadger.js

```diff
@@ -0,0 +1,16 @@
+import Honeybadger from 'honeybadger-js';
+import log from './log';
```

Comment:
Oh snap. This is a big improvement - setting the things we want in `utils/log.js` 💯